packages/forma-36-react-components/src/components/Tooltip/Tooltip.tsx

```diff
@@ -150,6 +150,10 @@ export const Tooltip = ({
     }
   }, [closeOnMouseLeave, isHoveringTarget, isHoveringContent]);
 
+  useEffect(() => {
```

Comment:
Run this once to set the initial visibility of the Tooltip to True if applicable

Reply:
Did you try to make the default value of isVisible to be true?

It looks to me that this effect should be doing the same as setting the initial state to true, but because isVisible does not have a value than it does not work, but idk tbh

Reply:
@gui-santos the initial state `const [show, setShow] = useState(isVisible);` alone doesn't work because `show` state is set again [here](https://github.com/contentful/forma-36/blob/42f6e513330980afa8fc1dc8ac7c6fe542a641bd/packages/forma-36-react-components/src/components/Tooltip/Tooltip.tsx#L147) and [here](https://github.com/contentful/forma-36/blob/42f6e513330980afa8fc1dc8ac7c6fe542a641bd/packages/forma-36-react-components/src/components/Tooltip/Tooltip.tsx#L149) to `false`. 
So I added another useEffect() block here, just to set the initial `show` state to the `isVisiable` value

Reply:
I see! Thanks for explaining it ✌️